Setup/UpgradeSchema.php

```diff
@@ -86,5 +86,42 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
 
             $installer->endSetup();
         }
+
+
+        if (version_compare($context->getVersion(), '1.1.0') < 0) {
+            $installer = $setup;
+            $installer->startSetup();
+
+            /**
+             * Update table 'tax_class'
+             */
+            $installer->getConnection()->addColumn(
+                $installer->getTable('tax_class'),
+                'tj_salestax_exempt_type',
+                [
+                    'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                    'default' => 'non_exempt',
```

Comment:
If a Magento merchant already has "TaxJar Exempt" set to **Yes** for an existing customer tax class and upgrades to this version, will they have to configure the tax class again to make it exempt? Do we need to show a notice, create a one-time migration, or something else?

Reply:
We could pick a default otherwise they would need to go in and resave the tax class or customer class. The default is non_exempt so unless they resave the admin form it will stay in that state. Perhaps a notice? What do you think?